### ISSUE TITLE
A11y_code.visualstudio.com_VisualStudioCode_Download_Platform Specific_ScreenReader: screen reader does not read all the text present on link 'Python, C/C++....' .

### ISSUE THREAD

**Issue by bhagwatshelke1047** [microsoft/vscode-docs#3721]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS1.3.1;#Win10;#Visual Studio Code;#Narrator:#NVDA;#CAITestPass_May2020;#FTP;#STP;#Chrome.

**Environment Details:**
Application Name: code.visualstudio.com
#URL:[https://code.visualstudio.com](url)
Windows Version: Windows10
Browser version: Microsoft Edge 84.0.488.1 (Official build) dev (64-bit), Google Chrome 81.0.4044.138 (Official Build) (64-bit)
Screen reader : Narrator, NVDA

**Repro Steps:**
1. Launch the application Url :[https://code.visualstudio.com](url)
2. Navigate to "Download" button.
3. Hit on any of the platform(Windows/Linux/MacOS) links or any installer type(*64 and *32).
5. Start screen reader and for links Python, C/C++..etc.

**Actual:**
Screen reader does not read all the text available on links 'Python, C/C++...etc'. It only reads 'Python link, C/C++ link' but does not read the description about the link provided on the link.
Ex.For Python screen reader only reads 'Python link'.

**This issue is observed with both the screen readers narrator and NVDA.**
**Same issue in the following page:**
1)Docs Get Started-->Themes
2)Docs User Guide-->IntelliSense

**Expected:**
Screen reader should read all the text available on the links. 
ex. For Python link it should read 'Python Ms-Python 20.5 M downloads Linting  debugging Multithreaded...etc link'.

Attachment for Reference:
https://microsoft-my.sharepoint.com/:v:/p/v-rojad/EWxcqVYjVtdNrVUYMCUUuTMBc0k9QDfm2X3dvprCmYDDcA?e=kEddzb

**Comment by fsteffi** [comment]
issue fixed with NVDA:https://code.visualstudio.com/docs/?dv=linux64_deb

severity can be downgraded as issue repro with narrator but same issue there when link opened 

**GithubTags:**#Regressed:02-03-23;
[3721.webm](https://user-images.githubusercontent.com/24580570/216558820-56a1bc9f-8152-4f40-bf5d-58a3f159899d.webm)

**Comment by Samhitha66** [comment]
Regressed the issue in the below environment
URL:https://code.visualstudio.com/docs/?dv=win64
Narrator is reading all the information present on links like Python , c++
The issue is fixed. Hence closing this bug.
#Regressed:08-08-23; #Closed;
![image](https://github.com/microsoft/vscode-docs/assets/99243079/81b372c2-6617-47af-a1be-c9ebf530d44f)
